lib/wasix/src/fs/fd.rs

```diff
@@ -97,6 +115,15 @@ pub enum Kind {
         /// Reference to the pipe
         pipe: Pipe,
     },
+    Epoll {
+        // List of events we are polling on
```

Comment:
Can we move this out to a separate type, to also keep logic sort of constrained in methods?